src/modules/billing/jobs/check-for-updated-invoice-accounts.js

```diff
@@ -26,9 +28,15 @@ const handler = async () => {
 
   // Call CRM to identify the invoice accounts
   const invoiceAccountsWithUpdatedEntities = await invoiceAccountsConnector.fetchInvoiceAccountsWithUpdatedEntities();
+  const templateId = config.notify.templates.nald_entity_changes_detected;
+  const recipient = process.env.NALD_SERVICE_MAILBOX;
 
-  // Publish a job which will call CM with each of the invoice account IDs
-  invoiceAccountsWithUpdatedEntities.map(invoiceAccount => messageQueue.getQueueManager().add(jobNames.updateCustomerAccount, invoiceAccount.invoiceAccountId));
+  if (invoiceAccountsWithUpdatedEntities.length > 0 && recipient) {
+    const content = invoiceAccountsWithUpdatedEntities.map(invoiceAccount => `
```

Comment:
It looks like `content` is an array here - is Notify happy accepting an array?

Reply:
Yes. It renders the array as bulletpoints :)